inc/plugin.class.php

```diff
@@ -1356,7 +1356,7 @@ static function doHook($name, $param = null) {
       } else { // Standard hook call
          if (isset($PLUGIN_HOOKS[$name]) && is_array($PLUGIN_HOOKS[$name])) {
             foreach ($PLUGIN_HOOKS[$name] as $plugin_key => $function) {
-               if (!Plugin::isPluginActive($plugin_key)) {
+               if (!Plugin::isPluginActive($plugin_key) && $name != "post_plugin_disable") {
```

Comment:
```suggestion
               if (!Plugin::isPluginActive($plugin_key) && !preg_match('/^post_plugin_[az]+$', $name)) {
```

Reply:
I guess it would be too large. It can really make sense if plugin is diabled; but bypassing the isActive should not be usefull in other cases